bpm/src/main/java/org/jboss/pnc/bpm/model/RepositoryManagerResultRest.java

```diff
@@ -16,43 +16,36 @@
  * limitations under the License.
  */
 
-package org.jboss.pnc.rest.restmodel;
+package org.jboss.pnc.bpm.model;
 
+import lombok.AllArgsConstructor;
 import org.jboss.pnc.model.Artifact;
 import org.jboss.pnc.spi.coordinator.CompletionStatus;
 import org.jboss.pnc.spi.repositorymanager.RepositoryManagerResult;
 
 import javax.xml.bind.annotation.XmlRootElement;
 import java.io.Serializable;
 import java.util.List;
-import java.util.stream.Collectors;
 
 /**
  * @author <a href="mailto:matejonnet@gmail.com">Matej Lazar</a>
  */
+@AllArgsConstructor
 @XmlRootElement(name = "repositoryManagerResult")
 public class RepositoryManagerResultRest implements Serializable {
-    private List<ArtifactRest> builtArtifacts;
-    private List<ArtifactRest> dependencies;
+    private List<org.jboss.pnc.dto.Artifact> builtArtifacts;
+    private List<org.jboss.pnc.dto.Artifact> dependencies;
```

Comment:
is JIRA for verification/fix in maitai created?

Reply:
On it!

Reply:
NCL-5052 created